windows/pyinstaller.spec

```diff
@@ -15,4 +15,4 @@ exe = EXE(pyz,
           strip=None,
           upx=True,
           console=False,
-          icon='plover.ico')
+          icon=os.path.join('plover/assets', 'plover.ico'))

```

Comment:
`windows/plover.ico` should be removed, and `windows/helper.py` updated instead:
```diff
diff --git i/windows/helper.py w/windows/helper.py
index f7b1eb2..a68ae21 100755
--- i/windows/helper.py
+++ w/windows/helper.py
@@ -32,7 +32,7 @@ from plover import (

 APPNAME = __software_name__.capitalize()
 VERSION = __version__
-ICON = os.path.join(WIN_DIR, '%s.ico' % __software_name__)
+ICON = os.path.join(TOP_DIR, __software_name__, 'assets', '%s.ico' % __software_name__)


 if sys.stdout.isatty():
```

I'll remove `windows/pyinstaller.spec` in a subsequent [pull request](https://github.com/openstenoproject/plover/compare/openstenoproject:12b554d...benoit-pierre:ce3335f).